modules/ibi-ib-cli-reference.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-image-based-install.adoc 
+
+:_content-type: REFERENCE
+[id="ibi-ib-cli-reference_{context}"]
+= ib-cli CLI tool reference
+
+The following matrix shows the command line arguments for `ib-cli` CLI tool. 
+
+.ib-cli CLI reference
+[options="header"]
+[cols="2,1,4"]
+|====
+|Argument|Optional|Description
+|`seed-image`|no|Specifies the seed image to use in the ISO generation process.
+|`seed-version`|no|Specifies the {product-title} release version of the seed image.
+|`auth-file`|no|Specifies the path to the authentication file of the container registry for the seed image. This file is used for authentication during the seed image pull process.
+|`pullsecret-file`|no|Specifies the path to the pull secret file used during the precache process. The pull secret contains authentication credentials for pulling the release payload images from the container registry.
+|`ssh-public-key-file`|yes|Specifies the path to the SSH public key file to be added to the installed host. This key allows SSH access to the host when booted with the installation ISO.
+|`lca-image`|yes|Specifies the image of the {lcao} Operator to be used for flushing the seed image to disk. The default value is `quay.io/openshift-kni/lifecycle-agent-operator:latest`.
+|`rhcos-live-iso`|yes|Specifies the URL to the {op-system} live ISO image used for generating the installation ISO.
+|`installation-disk`|no|Specifies the disk that will be used for the installation process. The installed host's operating system will be installed on this disk.
+|`extra-partition-start`|yes|Specifies the start of the extra partition used for `/var/lib/containers`. The partition will expand until the end of the disk. Uses sgdisk notation.
```

Comment:
The `extra-partition-start` argument is now mandatory

Reply:
Updated.